app/controllers/kittens_controller.rb

```diff
@@ -7,4 +7,30 @@ def index
 
     render layout: true
   end
+
+  def new
+    # TODO
+    # @kitten = Kitten.new
```

Comment:
Yes of course, but isn't this kind of intrusive? This migration would be part of plugin deployment (in addition to updating the gemfile.plugins) meaning that some extra scripts would have to be run in order to get things up and running.